mp4parse/src/boxes.rs

```diff
@@ -3,6 +3,16 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 use std::fmt;
 
+// To ensure we don't use stdlib allocating types by accident
```

Comment:
Out of interest, is there a term for this strategy to prevent usage?

Reply:
Not to my knowledge. I suppose you could say it's a form of shadowing, but this is the first time I've ever _wanted_ to prevent the use of types that are in the stdlib (and most of them in the prelude).